### PR TITLE
fix: add check_title_template to action inputs

### PR DESCRIPTION
**Pull request by MIJOTHY**
`check_title_template` is supported internally, but not exposed via the
action interface. This commit exposes it and adds an example usage to
the `build` job.

Relates to:
7a628808eaca1f93598d5226121fe2f638ffa400

### CODE REVIEW

**Comment by mikepenz** [comment]
Thank you very much for the PR. I am only partially available the next 14 days, so it will take me a bit longer to review.

**Review by mikepenz — APPROVED** [review]

**Comment by MIJOTHY** [comment]
Fixed a misplaced period in the description string in https://github.com/mikepenz/action-junit-report/pull/546/commits/0cbde44600a7b5a453264333cf0f01ba0b2cba2c after seeing the workflow failure

**Comment by mikepenz** [comment]
Given the failure: https://github.com/mikepenz/action-junit-report/runs/6192680363?check_suite_focus=true#step:4:1 
The `action.yml` does not allowed to hold a template

**Comment by MIJOTHY** [comment]
> Given the failure: https://github.com/mikepenz/action-junit-report/runs/6192680363?check_suite_focus=true#step:4:1 The `action.yml` does not allowed to hold a template

I can think of two options:
1. Remove the template vars from the docstring
2. Change the code to use a different templating pattern 

I'm leaning towards (2), as perhaps it's more user-friendly to avoid the clash with the GHA template delimiters. Currently a user would need to escape the `${{FILE_NAME}}` template in order to avoid it being interpreted as a GHA template, so usage would look like
```yaml
check_title_template: ${{ '${{FILE_NAME}}' }}
```
whereas if we were to change the action-internal template delimiters, we could have
```yaml
check_title_template: '${FILE_NAME}'
```

I think this wouldn't be breaking, as currently the option isn't exposed via the action in the first place. What are your thoughts?

**Comment by mikepenz** [comment]
Possibly a very different template needs to be chosen, as `${x}` aligns with the template commonly used in all sorts of languages, and I'd prefer to not run into problems due to this. 

Theoretically dropping the `$` should resolve the issue I suppose. 

E.g. changing the templates to `{{FILE_NAME}}`

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -22,6 +22,10 @@ jobs:
           check_name: Example JUnit Test Report
           report_paths: '**/surefire-reports/TEST-*.xml'
           summary: '<table><thead><tr><th> Application (src/applications) </th></tr></thead><tbody><tr><td> test </td></tr></tbody></table>'
+          # Since the template delimiters used internally clash with those
+          # used by GitHub Actions, escape them by wrapping them in a GHA
+          # string expression
+          check_title_template: ${{ '${{SUITE_NAME}} | ${{TEST_NAME}}' }}
```
Comment (mikepenz) [inline .github/workflows/build.yml]: Given the template change, could you also please update this part

**Review by mikepenz — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (MIJOTHY) [reply .github/workflows/build.yml]: Aha good spot, I missed that.

Comment (MIJOTHY) [reply .github/workflows/build.yml]: Fixed in https://github.com/mikepenz/action-junit-report/pull/546/commits/888386a4202477addec901dce17fc417bea38f9b

**Review by mikepenz — APPROVED** [review]

**Comment by mikepenz** [comment]
Thank you so much again @MIJOTHY

**Pull request merged**